### PR TITLE
Swiftify Geometry and NSView+RSCore

### PR DESCRIPTION
**Pull request by Wevah**
Issues #27 and #28.

### CODE REVIEW

File: RSCore/NSView+RSCore.swift [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	- Parameter subview: The subview to constrain. Must be a descendant of `self`.
+	*/
+	func addFullSizeConstraints(forSubview subview: NSView) {
+		let d = ["subview": subview]
+
+		var constraints = NSLayoutConstraint.constraints(withVisualFormat: "|[subview|", options: [], metrics: nil, views: d)
+		constraints.append(contentsOf: NSLayoutConstraint.constraints(withVisualFormat: "V:|[subview|", options: [], metrics: nil, views: d))
+
+		self.addConstraints(constraints)
+	}
```
Comment (kielgillard) [inline RSCore/NSView+RSCore.swift:38]: Is there an advantage or difference of the code above to the following code?
```
NSLayoutConstraint.activate([
	subview.leadingAnchor.constraint(equalTo: leadingAnchor),
	subview.topAnchor.constraint(equalTo: topAnchor),
	subview.trailingAnchor.constraint(equalTo: trailingAnchor),
	subview.bottomAnchor.constraint(equalTo: bottomAnchor)
])
```
If there is no difference, I suggest using anchors in this case better expresses intent (full sizing is achieved by the equality of anchors), code concision and resilience (compile time checking of anchors vs visual format correctness, fragile identifiers) and more efficient (no string parsing and array resizing).

If I was to guess at one advantage the visual format can give is that the system can create a comprehensive set of constraints for the expressed layout, constraints which layout system could use to optimise layout.

Comment (Wevah) [reply RSCore/NSView+RSCore.swift:38]: Nope, I was just porting what was in the old Objective-C version directly. I actually meant to switch to anchors and then promptly forgot.

Comment (brentsimmons) [reply RSCore/NSView+RSCore.swift:38]: I like @kielgillard’s suggestion — let’s switch to anchors here.

Comment (Wevah) [reply RSCore/NSView+RSCore.swift:38]: Agreed; I’ll make the change once I get back to my desk.

**Pull request merged**